reseq_ros2/reseq_ros2/scaler.py

```diff
@@ -0,0 +1,95 @@
+import rclpy
+from rclpy.node import Node
+import yaml
+from yaml.loader import SafeLoader
+# from reseq_interfaces.msg import Remote
+from geometry_msgs.msg import Twist
+from reseq_interfaces.msg import Remote, EndEffector
+import constants as rc
+"""
+ROS node that handles scaling of the remote controller data into physical variables used
+by the motors
+
+It receives a packet from the remote controller and rescales end_effector data 
+(pitch, head_pitch, head_yaw) and the Twist data used by Agevar 
+(linear velocity, angular velocity)
+"""
+
+class Scaler(Node):
+    def __init__(self):
+        super().__init__("scaler")
+        # TODO: read file passed as ROS argument
+        with open("src/reseq_ros2/reseq_ros2/config.yaml") as f:
+            self.config = yaml.load(f, Loader=SafeLoader)
+            print(self.config)
+
+        self.create_subscription(
+            Remote,
+            "/remote",
+            self.remote_callback,
+            10
+        )
+
+        self.enea_pub = self.create_publisher(
+            EndEffector,
+            "/end_effector",
+            10
+        )
+
+        self.agevar_pub = self.create_publisher(
+            Twist,
+            "/cmd_vel",
+            10
+        )
+
+        self.get_logger().info("Scaler node started")
+
+    def remote_callback(self, data: Remote): 
+        #TODO: buttons, switches
+        
+        cmd_vel = Twist()
+        cmd_vel.linear.y = data.right.y # Linear velocity (-1:1)
+        cmd_vel.angular.z = - data.right.x # Radius of curvature (-1:1)
```

Comment:
Don't we need some formula to convert the movement of the joystick into an angular velocity around Z?